fitbenchmarking/fitbenchmarking
# Adding jinja2 to installation

#### Description of Work

Fixes #324 

Adds jinja2 to the installation in `setup.py`

#### Testing Instructions

1. Check that tables produced on clean machine

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
